src/test/java/e2e/pages/BookPage.java

```diff
@@ -0,0 +1,60 @@
+package e2e.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.ElementClickInterceptedException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.testng.Assert;
+import util.Scroll;
+
+public class BookPage extends PageBase {
+    public BookPage(WebDriver driver) {
+        super(driver);
+    }
+
+    @FindBy(xpath = "//*[@class='text-right fullButton']//*[@id='addNewRecordButton']")
+    WebElement addToYourCollectionButton;
+
+    @FindBy(xpath = "//*[@class='element-list collapse show']//li[@id='item-3']//*[@class='text']")
+    WebElement profileButton;
+
+    @FindBy(xpath = "//*[@class= 'books-wrapper']")
+    WebElement bookInfoForm;
+
+    public void waitForLoadingBookInfoForm() {
+        getWait().forVisibility(bookInfoForm);
+        getWait().forClickable(bookInfoForm);
+    }
+
+    public void scrollToProfileButton() {
+        Scroll.scrollToElement(driver, profileButton);
+        getWait().forVisibility(profileButton);
+        getWait().forClickable(profileButton);
+    }
+
+    public void waitForLoadingProfileButton() {
+        getWait().forVisibility(profileButton);
+        getWait().forClickable(profileButton);
+    }
+
+    public void clickAddToYourCollectionButton() {
+        try {
+            addToYourCollectionButton.click();
+        } catch (ElementClickInterceptedException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public void clickProfileButton() {
+        profileButton.click();
+    }
+
+    public String checkThatSelectedBookOpened() {
```

Comment:
need to change this method , method do 1 thing , getTitleBook

Reply:
public String getTitleBook(){
   return element.getText()
}

Reply:
fixed